Hakawai/Mentions/HKWMentionsPluginV2.m

```diff
@@ -905,9 +909,23 @@ - (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range r
         self.currentlySelectedMentionRange = NSMakeRange(NSNotFound, 0);
     }
 
-    // Remove any mention selection that still exists, since it should go away upon any text change
-    [self toggleMentionsFormattingIfNeededAtRange:self.currentlySelectedMentionRange selected:NO];
-    self.currentlySelectedMentionRange = NSMakeRange(NSNotFound, 0);
+    // If more than one character is inserted
+    if (text.length > 0 && range.length > 0) {
+        // Remove any current selections
+        [self toggleMentionsFormattingIfNeededAtRange:self.currentlySelectedMentionRange selected:NO];
+        self.currentlySelectedMentionRange = NSMakeRange(NSNotFound, 0);
+
+        // Bleach a mention if the insertion intersects with it
+        // This is needed if a user autocorrects a mention name
```

Comment:
let's elaborate more on what autocorrects means here (i.e: black pop up menu with autocorrect options above text) so we know how to get here